attributes/default.rb

```diff
@@ -119,6 +120,10 @@
 default['hadoop_spark']['tf_spark_connector_version']                    = "2.11-1.12.0"
 default['hadoop_spark']['mysql_driver']                                  = "#{node['download_url']}/mysql-connector-java-5.1.29-bin.jar"
 
+
+default['hadoop_spark']['hsfs']['version']                          = node['install']['version']
```

Comment:
Since it follows the hopsworks version, does the release script need to release hsfs as well?

Reply:
Yes, I was going to open a PR there as well.